server/src/yml-objects/AbstractYmlObject.ts

```diff
@@ -86,11 +86,12 @@ export abstract class AbstractYmlObject implements CompletionItem {
     public enrichWith(
         documentation: string,
         type: string,
-        sourceElementName?: string,
+        sourceElement?: AbstractYmlObject,
         scopeStartOffset?: number,
         scopeEndOffset?: number,
     ): void {
-        this.sourceElementName = sourceElementName ?? 'static';
+        this.sourceElement = sourceElement ?? null;
```

Comment:
Weird line xD So if a value is given, you… use null instead or something? :thinking:

Reply:
Damn. Should be
```suggestion
        this.sourceElement = sourceElement;
```

Reply:
It seems that you did not edit this, nor did you apply your own suggestion.